Troubleshooting.md

```diff
@@ -0,0 +1,91 @@
+# Troubleshooting
+
+Some possible issues you may come across in trying to get Montagu running, especially on a local dev environment, and how to get around them. 
+
+Please add to this doc if you find and fix any similar issues. 
+
+
+## Docker Issues
+
+**'No basic auth credentials' error when fetching image from docker**
+
+You need to make sure you're logged in to docker as the montagu user, not as yourself, and for this you'll need to have vault set up. See here for some clues: https://github.com/vimc/montagu-registry/tree/master#login
+
+If you've been running montagu applications at all this *should* all be set up, but it went snafu for me somehow.
+
+
+**I've got a load of docker containers hanging around, how do I get rid of them all?**
+
+Run:
+```
+docker stop $(docker ps -aq)
+docker system prune --force
+```
+
+This script is present in some repos as `clear-docker.sh`.
+
+To get rid of old volumes, you can run
+```docker volume prune --force```
+
+To remove (but not stop) containers and volumes in one line, use ```docker system prune --force --volumes```. 
+
+
+**A container isn't starting or seems otherwise broken. How do I see its logs?**
+
+```
+docker logs [CONTAINER NAME]
+```
+
+## Vagrant / VM issues
+
+**How can I run a support VM locally?**
+
+From montagu-machine, run  `vagrant up <machine name>`
+
```

Comment:
The dev VM doesn't actually exist yet in this branch, but I'm working on it now in https://github.com/vimc/montagu-machine/tree/i2483